src/lib389/lib389/__init__.py

```diff
@@ -1048,24 +1048,6 @@ def close(self):
 
         self.state = DIRSRV_STATE_OFFLINE
 
-    def dump_errorlog(self):
```

Comment:
This helper function looks valid Are you sure you want to revert that change ?

Reply:
Yes, The fix is incomplete (a test is inverted) and had a side effect on CI tests 
 (because it change an exception type in some cases )
 I plan to create a clean PR for this improvement.